templates/_schemas/definitions/creativeWork.twig

```diff
@@ -0,0 +1,50 @@
+{% macro creativeWork(props, schema = null) %}
+  {% from '_schemas/definitions/thing' import thing %}
+  {% from '_schemas/definitions/organization' import organization %}
+  
+  {% set authors = [] %}
+  
+  {% for author in (props.authors ?? []) %}
+    {% if author.type is defined and author.type == 'person' %}
+      {% set authors = authors|merge([craft.schema.Person
```

Comment:
Let us create a macro for `Person` and call that? In this place as well as line 27.

Reply:
Done.